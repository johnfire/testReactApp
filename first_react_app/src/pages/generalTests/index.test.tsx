import { describe } from "node:test";
import renderer from "react-test-renderer";

import Index from "../index";

describe("general test suite warm up ", () => {
  test("adds 1 + 2 to equal 3", () => {
    expect(1 + 2).toBe(3);
  });

  test("prints a string", () => {
    console.log("here is a sting ");
  });
});

describe("general snapshots", () => {
  test("checks main index for sameness", () => {
    const tree = renderer.create(<Index />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
