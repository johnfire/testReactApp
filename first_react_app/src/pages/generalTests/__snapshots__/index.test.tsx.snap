// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`checks main index for sameness 1`] = `
<main
  className="main"
>
  <div
    className="description"
  >
    <div>
       my test component what is going on here where is the text 
    </div>
  </div>
</main>
`;
